src/_playground/Routes.js

```diff
@@ -270,59 +271,99 @@ const routes = [
     }
 ];
 
-export const Routes = () => {
-    let sectionRoutes;
-    const groupedRoutes = groupArray(routes, 'section');
+export class Routes extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            query: '',
+            filteredItems: routes
+        };
+    }
 
-    const navItems = sections.sort(sortBy('sortOrder', 'name')).map(section => {
-        if (!groupedRoutes[section.name]) {
-            return;
-        }
+    onChangeHandler = (event) => {
+        let searchResults = routes.filter((navItem) => {
+            return navItem.sortOrder === 1 ? navItem : navItem.name.toLowerCase().includes(event.target.value.toLowerCase());
```

Comment:
Thanks @greg-a-smith . Actually was trying to implement the spacing with sass, but it seems to be overwritten with that component's (using the inputgroup component) styling.

Re: the search, I believe this is purely meant to be a component search (like in the fundamental-ngx documentation). Thanks for the comment, will work on filtering be section instead of order. 👍